oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/mysql/MySQLTableInstaller.java

```diff
@@ -111,7 +111,7 @@ protected void createIndexes(JDBCHikariCPClient client, Model model) throws Stor
             case SERVICE_INVENTORY:
             case SERVICE_INSTANCE_INVENTORY:
             case NETWORK_ADDRESS:
-            case ENDPOINT_INVENTORY:
+            case ENDPOINT_TRAFFIC:
                 createInventoryIndexes(client, model);
```

Comment:
`EndpointTraffic` is not `RegisterSource` type, but creating `InventoryIndexes` needs a `sequence` column, which is defined in `RegisterSource`

Reply:
Removed.